libraries/AP_Telemetry/VersionInfo.h

```diff
@@ -0,0 +1,7 @@
+#ifndef VERSIONINFO_H
+#define VERSIONINFO_H
+
+#define BUILD_TIMESTAMP "2017/04/20 15:54:48"
```

Comment:
@muramura Thank for your comment. 
I delete this file because it is not necessary. This file is created at the timing of configure and the timestamp is automatically entered.